### DISCUSSION TITLE
Verify Google Play Subscription 403 Forbidden

### DISCUSSION THREAD

**Discussion by mretcoyy** [imdhemy/laravel-in-app-purchases#176] [Q&A] [answered]
I already input package name on .env
$itemId = google play json response (productId)
token =  purchaseToken.

**Comment by imdhemy** [comment, marked as answer]
I believe this is related to setting up the credentials. You can find more information in the documentation.

https://imdhemy.com/laravel-iap-docs/docs/credentials/google-play

**Reply by mretcoyy** [reply]
Hi imdhemy
I already got the json file. but when I put the file location on .ENV
This error message is what I've got.

"Could not load the default credentials. Browse to https://developers.google.com/accounts/docs/application-default-credentials for more information"

**Reply by imdhemy** [reply]
I will check it and get back to you

**Reply by imdhemy** [reply]
Duplication #140
